packages/flutter_tools/lib/src/device.dart

```diff
@@ -35,10 +36,16 @@ class DeviceManager {
     IOSSimulators(),
     FuchsiaDevices(),
     FlutterTesterDevices(),
-    MacOSDevices(),
-    LinuxDevices(),
-    WindowsDevices(),
-  ]);
+  ] + _maybeAddDesktop);
```

Comment:
I did not know that the additional operator worked for lists...

Reply:
I think it was added in 2.0 or shortly thereafter